lib/code_corps/policy/organization_github_app_installation.ex

```diff
@@ -6,9 +6,19 @@ defmodule CodeCorps.Policy.OrganizationGithubAppInstallation do
 
   alias CodeCorps.{OrganizationGithubAppInstallation, User}
 
-  def create?(%User{} = user, %Ecto.Changeset{} = changeset),
-    do: changeset |> get_organization |> owned_by?(user)
+  def create?(%User{} = user, params) do
+    params |> atomize_params |> get_organization |> owned_by?(user)
+  end
 
-  def delete?(%User{} = user, %OrganizationGithubAppInstallation{} = github_app_installation),
-    do: github_app_installation |> get_organization |> owned_by?(user)
+  def delete?(%User{} = user, params),
+    do: params |> atomize_params |> get_organization |> owned_by?(user)
```

Comment:
You can completely revert this change if you fix the controller action.

Reply:
updated